src/Microsoft.DotNet.Compiler.Common/ProjectContextExtensions.cs

```diff
@@ -34,25 +37,40 @@ public static void MakeCompilationOutputRunnable(this ProjectContext context, st
 
             if (context.TargetFramework.IsDesktop())
             {
+                // On full framework, copy all dependencies to the output path
                 exporter
                     .GetDependencies()
                     .SelectMany(e => e.RuntimeAssets())
                     .CopyTo(outputPath);
+                    
+                // Generate binding redirects
+                var outputName = context.GetOutputPathCalculator(outputPath).GetAssemblyPath(configuration);
```

Comment:
This is a bug. This will add config/tfm again on a path already with config/tfm. Need to figure out the best way to deal with this. Right now, the callers expect to pass the final path and not the base path.

I am debating push the output path calculator entirely in there, but it will be weird for build/compile to calculate paths and then pass base path again. And this will also change restore.

Another option is to add an overload to getAssemblyPath where you can pass the output path to it. I am leaning towards this one.

Opnions? After fixing this (which is causing the TestBindingRedirectGeneration, good testing), then I will move the logic over to build.